packages/notification-services-controller/CHANGELOG.md

```diff
@@ -7,11 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.1]
+
+### Changed
+
+- `NotificationServicesController` mobile fixes ([#4441](https://github.com/MetaMask/core/pull/4441))
```

Comment:
Looks like there are feature additions here that might make this a minor release.

```suggestion
## [0.2.0]

### Added

- Export `NOTIFICATION_CHAINS_ID`, which is a const-asserted version of the `NOTIFICATION_CHAINS` object. ([#4441](https://github.com/MetaMask/core/pull/4441))
- Export const-asserted objects `NOTIFICATION_NETWORK_CURRENCY_NAME`, `NOTIFICATION_NETWORK_CURRENCY_SYMBOL`. ([#4441](https://github.com/MetaMask/core/pull/4441))
- Export `defaultState` object from `NotificationServicesPushController` ([#4441](https://github.com/MetaMask/core/pull/4441))

### Fixed

- `NotificationServicesController` mobile fixes ([#4441](https://github.com/MetaMask/core/pull/4441))
```

- Could we expand the "Fixed" entry to explain what's being fixed and any user-facing behavior changes?
  - Does the logic around `#isPushIntegrated` cause user-facing changes?
  - Is the replacement of `sw` with `self` a user-facing change?
- Is the removal of exports from `mockServices` user-facing? If so, it should be listed under a `### Removed` heading.

Reply:
TY for comments, I've added a more descriptive changelog